plugins/procfs/README.md

```diff
@@ -1,3 +1,153 @@
+|`mem_available`|Gauge|bytes|More accurate estimate of the actual available memory, taking into account free memory and also caches and buffers that can be freed|LocalMachine|LocalMachine||
+|`cached`|Gauge|bytes|Cached is the memory used for ROM cache (page cache) in memory|LocalMachine|LocalMachine||
+|`swap_cached`|Gauge|bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
+
+### Attributes
+
+#### Kind
+
+The kind of the memory is the allocated memory space reserved by the system or the hardware (<https://man7.org/linux/man-pages/man5/proc_pid_status.5.html>) :
+
+|Value|Description|
+|-----|-----------|
+|`resident`|Resident set size, measured in pages (same as VmRSS in `/proc/<pid>/status`)|
+|`shared`|number of resident shared pages (i.e., backed by a file) (same as RssFile+RssShmem in `/proc/<pid>/status`)|
+|`virtual`|Virtual memory size (same as VmSize in `/proc/<pid>/status`)|
+
+The kind of the CPU time delta is the average CPU time spent by various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time spent by the process in user mode|
+|`system`|CPU time used by the system (Kernel)|
+|`guest`|CPU time used by a guest virtual machine when a hypervisor virtualizes environments|
+
+#### cpu_state
+
+The CPU states correspond the amount of time, measured in ticks, the CPU has been in specific states. These fields are measured in ticks because the underlying data from the kernel is measured in ticks (<https://docs.rs/procfs-core/latest/procfs_core/struct.CpuTime.html>) :
+
+|Value|Description|
+|-----|-----------|
+|`user`|Ticks spent in user mode|
+|`nice`|Ticks spent in user mode with low priority (nice)|
+|`system`|Ticks spent in system mode|
+|`idle`|Ticks spent in the idle state|
+|`irq`|Ticks servicing interrupts|
+|`softirq`|Ticks servicing soft interrupts|
+|`steal`|Ticks of stolen time. Stolen time is the time spent in other operating systems when running in a virtualized environment.|
+|`guest`|Ticks spent running a virtual CPU for guest operating systems under control of the linux kernel|
+|`guest_nice`|Ticks spent running a niced guest|
+
+## Configuration
+
+Here is a configuration example of the plugin. It's part of the ALUMET configuration file (eg: `alumet-config.toml`). Its configuration is composed of different sections and each of them can be enabled or disabled with the `enabled` parameter to be set to true or false.
+
+To active the plugin to collect metrics relative to the kernel utilization :
```

Comment:
Can you create one markdown subsection for each sub-config? `### Kernel metrics` for instance